generated/nidmm/session.py

```diff
@@ -186,6 +196,7 @@ def __init__(self, resource_name, id_query=0, reset_device=False, options_string
         self.library = library.get_library()
         self.vi = 0  # This must be set before calling _init_with_options.
         self.vi = self._init_with_options(resource_name, id_query, reset_device, options_string)
+        self.acquisition = Acquisition(self)
```

Comment:
Yes. That is what I had originally but it looks odd because of the redundancy:
```
with nidmm.Session('dev1') as session:
    with session.Acquisition(session):
        print(session.fetch(1000))
```
Note how session is in that line twice. If we are okay with that, then I can go back to that.

And do we need to return the Acquisition object? If so, why? What would we do with it?